tests/azure/availset_test.go

```diff
@@ -69,11 +69,11 @@ func TestAvailSetDeletion(t *testing.T) {
 
 	defer resources.Delete(t, resgroup)
 
-	availSets.Check(t, availset, resgroup)
+	availSets.AssertExists(t, availset, resgroup)
 
 	err = shell.Exec("TestAvailSetCreation", `
```

Comment:
hmm, I forgot to check the err :-)

But the `shell` symbol there is the `nash` object, not yours. I'll create a wrapper again but passing the shell too. Something like:

```go
func (nash *Nash) Exec(t *testing.T, shell *Shell, testName, content string)
```

and then use like this:

```go
    nash.Exec(t, "TestAvailSetCreation", `azure_something()`)
```